packages/api/amplify_api_dart/lib/src/api_plugin_impl.dart

```diff
@@ -21,18 +21,15 @@ import 'package:meta/meta.dart';
 class AmplifyAPIDart extends APIPluginInterface with AWSDebuggable {
   /// {@macro amplify_api_dart.amplify_api_dart}
   AmplifyAPIDart({
-    List<APIAuthProvider> authProviders = const [],
+    APIPluginOptions options = const APIPluginOptions(),
     ConnectivityPlatform connectivity = const ConnectivityPlatform(),
```

Comment:
Q: Was this missed?

Reply:
this is intended for the API flutter package to provide platform-specific implementations for ConnectivityPlatform, and it is not related to plugin options.